stable/influxdb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: influxdb
-version: 1.2.1
+version: 1.2.2
```

Comment:
Should we bump version to 1.3.0? The PR was a new feature and in a backward-compatible manner?

Reply:
true!